### ISSUE TITLE
DCU Web Source

### ISSUE THREAD

**Issue by VeNoMouS** [guessit-io/guessit#584]
Can we add `DC Universe` as a detected web streaming service for `DCU` enumeration field?

ie

```
Titans.2018.S01E01.Titans.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E02.Hawk.And.Dove.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E03.Origins.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E04.Doom.Patrol.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E05.Together.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E06.Jason.Todd.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E07.Asylum.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E08.Donna.Troy.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E09.Hank.And.Dawn.720p.DCU.WEB-DL.AAC2.0.H264-NTb
Titans.2018.S01E10.Koriandr.720p.DCU.WEB-DL.AAC2.0.H264-NTb
```

```
>>> import json
>>> import guessit
>>> guessit.__version__
'3.0.3'
>>> print json.dumps(guessit.guessit('Titans.2018.S01E01.Titans.720p.DCU.WEB-DL.AAC2.0.H264-NTb'), indent=4)
{
    "title": "Titans",
    "year": 2018,
    "season": 1,
    "episode": 1,
    "episode_title": "Titans",
    "screen_size": "720p",
    "source": "Web",
    "audio_codec": "AAC",
    "audio_channels": "2.0",
    "video_codec": "H.264",
    "release_group": "NTb",
    "type": "episode"
}
```

**Comment by ratoaq2** [comment]
Sure, probably we should also add streaming services to the advanced options, so the user can define them as needed

**Comment by aviadlevy** [comment]
Hi,
Can I open PR that add DCU to streaming-services?
I know that a small one, but you need to start somewhere ☺️

**Comment by ratoaq2** [comment]
Fixed by 9d422ccefaff77fc981d1781eb79de5420b91b79

I also made the streaming_service configurable through advanced options
